src/Trains.NET.Rendering.Skia/SKCanvasWrapper.cs

```diff
@@ -67,6 +68,47 @@ public void GradientRect(float x, float y, float width, float height, Color star
             _canvas.DrawRect(x, y, width, height, paint);
         }
 
+        public void GradientRectLeftRight(float x, float y, float width, float height, IEnumerable<Color> colours)
```

Comment:
I think we should probably consolidate all of these and add some kind of gradientdirection flag instead